.travis.yml

```diff
@@ -6,30 +10,117 @@ branches:
     only:
         - master
 
-language: cpp
-compiler:
-    - g++
-    - clang
-
-sudo: false
-
-addons:
-    apt:
-        sources:
-            - ubuntu-toolchain-r-test
-            - george-edison55-precise-backports
-        packages:
-            - g++-4.9
-            - clang
-            - cmake-data
-            - cmake
+matrix:
+  include:
+    - name: "default build (C++14)"
+      compiler: g++-8
+      env:
+        - ARCH=amd64
+        #compiler
+        - COMPILER=g++-8
+        - ACXX=g++-8
+        - ACC=gcc-8
+        #build settings
+        - build_type=Release
+        - SANITIZER=false
```

Comment:
I think we will now build C++11 by default, not C++14.
So this item should contain a line `CXX_STANDARD=14`